cms/appresolver.py

```diff
@@ -248,8 +266,11 @@ def _get_app_patterns(site):
         for lang in hooked_applications[page_id].keys():
             (app_ns, inst_ns), current_patterns, app = hooked_applications[page_id][lang]  # nopyflakes
             if not resolver:
+                regex_pattern = r''
+                if django.VERSION >= (2, 0):
```

Comment:
regex_pattern is being overriden, use an if / else or just an inline:
`regex_pattern = RegexPattern(regex_pattern) if django.VERSION >= (2, 0) else r''`